src/main/webapp/app/assessment-shared/external-submission/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './external-submission-button.component';
+export * from './external-submission.service';
+export * from './external-submission-dialog.component';
```

Comment:
Please don't create barrel files. We had so many problems with them in the past, that we decided to refrain from using them for new code.

Reply:
I removed the barrel file.